ui/components/component-library/header-base/index.ts

```diff
@@ -0,0 +1 @@
+export { HeaderBase } from './header-base';
```

Comment:
We could export `HeaderBaseProps` here as we will need it for `PopoverHeader` and `ModalHeader` but that could happen when it is needed